themes/default/register.tmpl.php

```diff
@@ -73,13 +73,13 @@ function encrypt_password()
 
 		<tr>
 			<td align="left"><div class="required" title="<?php echo _AC('required_field'); ?>">*</div><label for="login"><?php echo _AC('login_name'); ?></label>:</td>
-			<td align="left"><input id="login" name="login" type="text" maxlength="20" size="30" value="<?php if (isset($_POST['login'])) echo $stripslashes(htmlspecialchars($_POST['login'])); else echo $stripslashes(htmlspecialchars($this->user_row['login'])); ?>" /></td>
+			<td align="left"><input id="login" name="login" type="text" size="50" value="<?php if (isset($_POST['login'])) echo $stripslashes(htmlspecialchars($_POST['login'])); else echo $stripslashes(htmlspecialchars($this->user_row['login'])); ?>" /></td>
 		</tr>
 
 		<tr>
 			<td align="left" colspan="2">
 				<small>&middot; <?php echo _AC('contain_only'); ?><br />
-					   &middot; <?php echo _AC('20_max_chars'); ?></small>
```

Comment:
If the term `20_max_chars` is not used anywhere else, this record should be removed from `langauge_text.sql` so it won't be inserted into the language table.